SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -530,6 +559,12 @@ void USpatialNetDriver::BeginDestroy()
 	{
 		Connection->SendDeleteEntityRequest(WorkerEntityId);
 	}
+
+#if WITH_EDITOR
+	// Ensure our deployment started delegate is removed when the net driver is shut down.
```

Comment:
I had trouble reading the comment initially.
```suggestion
	// Ensure our OnDeploymentStart delegate is removed when the net driver is shut down.
```